### PR TITLE
Add Node.js installation snippet.

### PR DESCRIPTION
**Pull request by gonzunigad**
Add Node.js installation snippet.

### CODE REVIEW

**Comment by ffiebig** [comment]
IMHO, I wouldn't remove the extra section inviting people to read the PHP SDK's README. PHP has different alternatives of installation and it kind of needs that section.

We can argue if Node needs that explanation or link, I would just mention that you can also use Yarn and then add a second line of code using yarn, it is straightforward enough

File: documentacion/como_empezar/README.md [hunk 1]
```diff
@@ -47,7 +47,15 @@ ejecutando esto en la línea de comandos cuando estés en la raíz de tu proyect
 composer require transbank/transbank-sdk
 ```
 
-Te recomendamos leer [las instrucciones de instalación detalladas para el SDK PHP](https://github.com/TransbankDevelopers/transbank-sdk-php#instalaci%C3%B3n) para mas opciones de instalación.
```
Comment (ffiebig) [inline documentacion/como_empezar/README.md:50]: I wouldn't delete this

File: documentacion/como_empezar/README.md [hunk 2]
```diff
@@ -47,7 +47,15 @@ ejecutando esto en la línea de comandos cuando estés en la raíz de tu proyect
...
+[En
+**Node.js**](https://github.com/TransbankDevelopers/transbank-sdk-nodejs#instalaci%C3%B3n)
+se debe instalar por NPM o Yarn, ejecutando esto en la línea de comandos cuando estés en la raíz de tu proyecto:
+
+```bash
+npm install transbank-sdk --save
```
Comment (ffiebig) [inline documentacion/como_empezar/README.md:55]: I'd add a second line for yarn, the other option is to leave the Readme's link in the next section, whatever you think is better

Comment (gonzunigad) [reply documentacion/como_empezar/README.md:58]: Done.

**Pull request merged**